apply-config.sh

```diff
@@ -129,6 +129,6 @@ cp coc-settings.json ~/.config/nvim/coc-settings.json
 cp myinit.lua ~/.config/nvim/myinit.lua
 cp vimrc ~/.vimrc
 
-if [ add_color = true ]; then
```

Comment:
I just realised it works that way. Maybe should I change all the conditions through the file ?